analyses/snv-callers/util/wrangle_functions.R

```diff
@@ -29,7 +29,8 @@ set_up_maf <- function(maf_df, metadata_df = NULL) {
   maf_df <- maf_df %>%
     dplyr::mutate(
       # Calculate the variant allele frequency
-      vaf = as.numeric(t_alt_count) / (as.numeric(t_ref_count) + as.numeric(t_alt_count)),
+      vaf = as.numeric(t_alt_count) / 
```

Comment:
Would it be more or less memory intensive if I split these mutate steps into separate things where they are re-assigned and so I can put messages in between?